miner/block_builder.go

```diff
@@ -402,12 +399,15 @@ func (t *BlockBuilder) acceptBlockData() {
 			return
 
 		case layerID := <-t.beginRoundEvent:
+			t.Info("builder got layer %v", layerID)
```

Comment:
we see released tick every layer this should be corresponding.. either debug or remote